pipenv/vendor/pipfile/api.py

```diff
@@ -62,6 +62,24 @@ def __init__(self, filename='Pipfile'):
     def __repr__(self):
         return '<PipfileParser path={0!r}'.format(self.filename)
 
+    def inject_environment_variables(self, d):
```

Comment:
Generally anything in the `vendor` directory needs to be kept pristine, any modifications would require this package to move into the `patched` directory

More significantly you should simultaneously PR this change back up to the pipfile repository as well since we are not the only project which uses pipfiles

Reply:
I did that here: https://github.com/pypa/pipfile/pull/105